assets/src/edit-story/app/uploader/test/useUploader.js

```diff
@@ -141,6 +141,38 @@ describe('useUploader', () => {
       );
     });
 
+    it('throws an error if file too large to transcode', async () => {
```

Comment:
This case is already covered by the identical `throws an error if file is too large for transcoding` test below.

What's missing is a case for `validateFileForUpload( { size: 1024 * 1024 * 150, type: 'video/mp4' }, true, false )` ("file is too large to upload but can be transcoded", the case this PR changes) and `validateFileForUpload( { size: 1024 * 1024 * 50, type: 'video/mp4' }, false, false )` ("file is not too large to upload").

Jus tinkering with this locally, the test cases for all scenarios would look like this:

```js
it('throws an error if file is too large for transcoding', async () => {
  const {
    actions: { validateFileForUpload },
  } = setup({
    maxUpload: 1024 * 1024 * 1024 * 10,
  });

  await expect(() =>
    validateFileForUpload(
      {
        size: 1024 * 1024 * 1024 * 3,
        type: 'video/quicktime',
      },
      true,
      true
    )
  ).toThrow(
    'Your file is too large (3072 MB) and cannot be processed. Please try again with a file that is smaller than 2048 MB.'
  );
});

it('does not throw an error if file is too large to upload but can be transcoded', async () => {
  const {
    actions: { validateFileForUpload },
  } = setup({});

  await expect(() =>
    validateFileForUpload(
      { size: 1024 * 1024 * 150, type: 'video/mp4' },
      true,
      false
    )
  ).not.toThrow();
});

it('throws an error if file is too large to transcode or upload', async () => {
  const {
    actions: { validateFileForUpload },
  } = setup({});

  await expect(() =>
    validateFileForUpload(
      { size: 1024 * 1024 * 1024 * 2, type: 'video/mp4' },
      false,
      true
    )
  ).toThrow(
    'Your file is 2048MB and the upload limit is 100MB. Please resize and try again!'
  );
});

it('does not throw an error if file is not too large to upload', async () => {
  const {
    actions: { validateFileForUpload },
  } = setup({});

  await expect(() =>
    validateFileForUpload(
      { size: 1024 * 1024 * 50, type: 'video/mp4' },
      false,
      false
    )
  ).not.toThrow();
});
```